docs/development/tooling.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "Tooling"
+date: 2022-01-28T00:00:00+00:00
+weight: 40
+geekdocRepo: https://github.com/owncloud/web
+geekdocEditPath: edit/master/docs/development
+geekdocFilePath: tooling.md
+---
+
+{{< toc >}}
+
+## Packaging
+
+TODO: some sentences about pnpm and vite
+
+## Development Setup
+
+TODO: some sentences and command line examples for docker-compose dev stack
```

Comment:
I think a lot can be copied/moved here from the testing docs (and then linked accordingly)